plugins/rikki/loungeviews/components/playoffoverview/default.htm

```diff
@@ -117,29 +117,25 @@ <h6>Team Description:</h6>
                                 <div class="record">
                                     <div class="opponent opponent-1">
                                         {%if match['model'].teams[0] %}
-                                        {%if match['model'].teams[0].logo is not null%}<div class="logo"><img src="{{match['model'].teams[0].logo.path | resize(64,64)}}" style="position: relative; padding-top:0px;"title="{{match['model'].teams[0].title}}"></div>{%endif%}
-                                        <div class="name">{{match['model'].teams[0].title}}</div>
-                                        {%else%}
-                                        <div class="name disabled">TBD</div>
-                                        {%endif%}
+                                        <div class="logo spoiler">{%if match['model'].teams[0].logo is not null%}<img src="{{match['model'].teams[0].logo.path | resize(64,64)}}" style="position: relative; padding-top:0px;"title="{{match['model'].teams[0].title}}">{%endif%}</div>
+                                        <div class="name spoiler">{{match['model'].teams[0].title}}</div>{%else%}<div class="name disabled">TBD</div>{%endif%}
```

Comment:
I would like to maintain the linebreaks here (bevore the else) so that it is clear which if it refers to